### PR TITLE
Delete event

### LINKED ISSUE #38: Delete an Event
User should be able to delete an event.

### PR DESCRIPTION
**Pull request by caromurp**
Recreating this PR (might be unnecessary).  Formatting from last delete event PR was not merged.

### CODE REVIEW

**Comment by allysmith12** [comment]
Looks like you need to merge main into this branch

File: mobile/src/events/eventsService.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as Yup from "yup";
 export const validateEventSchema = () => {
   return Yup.object().shape({
     name: Yup.string().required("Name required"),
-    formattedAddress: Yup.string().required("Location Required"),
+    //formattedAddress: Yup.string().required("Location Required"),
```
Comment (allysmith12) [inline mobile/src/events/eventsService.ts:9]: Uncomment this!

**Review by allysmith12 — APPROVED** [review]
Once you uncomment the formattedAddress part looks great

File: mobile/src/events/eventsService.ts (hunk 1 above)
Comment (caromurp) [reply mobile/src/events/eventsService.ts:9]: oooo nice catch thanks!!

**Pull request merged**